src/models/authorities/index.ts

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable */
```

Comment:
why did you disable eslint?

Reply:
![Screenshot_3](https://user-images.githubusercontent.com/71778107/129255400-78df3a71-8a8f-4493-91e4-a061d94f91e2.png)

Reply:
esLint disabled in all files index.ts at reducer